router/xgress_edge/fabric.go

```diff
@@ -77,6 +77,8 @@ func (self *edgeTerminator) close(notify bool, reason string) {
 			logger.Error("no controller available, unable to remove terminator")
 		} else if err := self.edgeClientConn.removeTerminator(ctrlCh, self); err != nil {
 			logger.WithError(err).Error("failed to remove terminator")
+		} else {
+			logger.Info("Successfully removed terminator on channel close")
```

Comment:
a. this really necessary?
b. if it's necessary, i feel like it should have the id in the message
c. i worry that this will be spammy and unhelpful without additional info

Reply:
This is moving the message from the deleted code to here, to maintain the existing logging messages. The logger has the terminator id in it, added at the top of the function.